lib/reality/list.rb

```diff
@@ -44,16 +44,18 @@ def initialize(*names)
     # as possible. Typically, when you want to load several entities
     #
     # @return [self]
-    def load!
-      compact.partition(&:wikidata_id).tap{|wd, wp|
-          load_by_wikipedia(wp)
-          load_by_wikidata(wd)
+    def load!(frame = 3)
```

Comment:
Not sure, may be this  `frame` vare should be configured somehow in general gem configurations?